### ISSUE TITLE
Python error when edit in place is active

### ISSUE THREAD

**Issue by Gustry** [NationalSecurityAgency/qgis-shapetools-plugin#40]
There is a python exception every time I use the Processing toolbox with "edit in place" active.

In CreateArcAlgorithm
```
    def outputWkbType(self, input_wkb_type):
        if self.shape_type == 0:
```

`shape_type` is not defined

**Comment by hamiltoncj** [comment]
@Gustry I got burned with a change in the QGIS API. For some reason QGIS decided to query all the outputWkbType methods from the processing algorithms. This does not occur with the LTR release, but only in the latest. I only was initializing shape_type when the user executed the algorithm. I have fixed the problem although I need to ask the QGIS developers why this is being called outside of the processing algorithm. Version 3.4.0 is the new release. Let me know if you see any other issues.
